src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  padding-inline: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: absolute;
+  top: 276px;
```

Comment:
Using absolute positioning for the stars container might cause issues with responsiveness. Consider using relative positioning or a layout model like flexbox or grid to ensure the stars section adjusts well on different screen sizes.